src/test/java/spoon/test/query_function/AllLocalVars.java

```diff
@@ -0,0 +1,263 @@
+package spoon.test.query_function;
+
+import static org.junit.Assert.assertTrue;
+
+import java.io.IOException;
+import java.io.Reader;
+import java.io.StringReader;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.lang.reflect.Modifier;
+import java.util.function.Consumer;
+
+/*
+ * The values of the fields must be a sequence starting from 1

```

Comment:
> OK. Basically, you check the behavior of Java itself.

I check that all the numbers which are later used in VariableReferencesTest are assigned well, following java behavior.

>  "JavaBehaviorTest", annotate methods with @Test

ok, sounds good.